### PR TITLE
Adding a more specific README in markdown for GitHub and DLT viewer.

### PR DESCRIPTION
**Pull request by jeremiah**
Signed-off-by: Jeremiah C. Foster <jeremiah@jeremiahfoster.com>

I'm sending this pull request even though I'm a member of the GENIVI GitHub Diagnostics team simply because I want to use this as a code review and to test pull requests. Please let me know if you would prefer another means of code review or receiving patches. 

This new file I'm sending as a pull request is to clarify the GitHub DLT Viewer repo and its usage, I hope its useful. I've updated URLs for example.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+https://at.projects.genivi.org/wiki/display/PROJ/Diagnostic+Log+and+Trace
+
+Mailinglist
+-----------
+https://lists.genivi.org/mailman/listinfo/genivi-diagnostic-log-and-trace
+
+Contact
+-------
+Alexander Wenzel <Alexander.AW.Wenzel@bmw.de> +
+Lutz Helwind <Lutz_Helwing@mentor.com>
```
Comment (clipka) [inline README.md:71]: Just wonder if Lutz is still involved ... Furthermore, Last name is misspelled.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
+Diagnostic Log and Trace - README.md
+
+Overview
+---
+The Diagnostic Log and Trace Viewer is an application,
+written for Windows and Qt, that can send and receive control messages
```
Comment (clipka) [inline README.md:6]: also for Linux (and Mac?)

File: README.md (hunk 1 above)
Comment (GeniviDLTmaintainer) [reply README.md:71]: Remove Lutz and add 
Gernot Wirschal Gernot.Wirschal@bmw.de
instead

File: README.md (hunk 2 above)
Comment (GeniviDLTmaintainer) [reply README.md:6]: Please change to:

based on Qt for Windows and Linux (optional Mac)

We test DLT Viewer only for Windows and Linux, Mac is tested by some users from time ton time.

**Pull request closed without merging**